core/app/models/concerns/spree/default_price.rb

```diff
@@ -23,6 +23,13 @@ def self.default_pricing
       end
     end
 
+    # Returns `#prices` prioritized for being considered as default price
+    #
+    # @return [Array<Spree::Price>]
```

Comment:
`currently_valid` is a regular scope so it returns an `ActiveRecord::Relation`, isn't it?

Reply:
Right! Fixed.